test/tests/functional/pbs_fairshare.py

```diff
@@ -79,6 +79,12 @@ def test_formula_keyword(self):
         formula = 'fairshare_factor'
         self.server.manager(MGR_CMD_SET, SERVER, {'job_sort_formula': formula})
 
+        formula = 'fair_share_factor'
+        try:
+            self.server.manager(MGR_CMD_SET, SERVER, {'job_sort_formula': formula})
+        except PbsManagerError, e:
```

Comment:
This is the old style (pre-python 2.6).  The new style is except PbsManagerError as e: